.github/workflows/build-test-smart-contracts.yaml

```diff
@@ -243,35 +208,25 @@ jobs:
           #Remaining example contracts have not yet been added to ci
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ env.RUST_CLIPPY }}
-          override: true
+        uses: actions/checkout@v4
```

Comment:
There is this comment `#Remaining example contracts have not yet been added to ci`. Just wondering if we could add the remaining tests.

Reply:
Yes, but I think it's not entirely trivial so I would like to postpone this to a separate PR.

I added an issue https://github.com/Concordium/concordium-base/issues/481